server/controllers/AuthorController.js

```diff
@@ -29,4 +29,52 @@ export default class AuthorController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+   * @description allows admin to update an author
+   * @param {object} request express request object
+   * @param {object} response express response object
+   * @returns {json} json
+   * @memberof AuthorControllers
+   */
+  static async updateAuthor(request, response) {
+    const { params: { authorId }, body: { authorName } } = request;
+    try {
+      const authors = await Author.findAll();
```

Comment:
I prefer the way you did it before. There is no need to get all the authors from the database at once. If there are very many authors in the database, this query would be very intensive. Use the findOne instead. That is better.